package.json

```diff
@@ -30,12 +26,12 @@
     "preinstall": "npm run npmcheckversion",
     "postinstall": "npm run build:dll",
     "prebuild": "npm run build:clean",
-    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js --color -p --progress --hide-modules --display-optimization-bailout",
+    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.dev.babel.js --color -p --progress --hide-modules --display-optimization-bailout",
```

Comment:
What is triggering the build process? Do we have to do it manually? If so, update the repo readme to include this step.